index.html

```diff
@@ -63,12 +63,43 @@ <h3>Code Journal</h3>
         <div class="entries-header">
           <h1>Entries</h1>
           <ul>
-            <li><a href="#entry-form">NEW</a></li>
+            <li><input class="button" type="button" id="new" value="NEW"></li>
           </ul>
         </div>
         <ul id="journal-entries">
         </ul>
       </div>
+      <div class="edit-entries hidden" data-view="edit-entries hidden" id="edit-entries">
+        <header>
+          <div class="sibling-header container">
+            <h3>Code Journal</h3>
+            <p>Entries</p>
+          </div>
+        </header>
+        <div class="entries-header">
+          <h1>Edit Entry</h1>
+        </div>
+        <form id="edit-form">
+          <div class="row">
+            <div class="column-half">
+              <img id="edit-image" src="images/placeholder-image-square.jpg">
+              <div class="two-rows">
+                <h3 class="title">Title</h3>
+                <input type="text" id="edit-title" name="title" required>
+                <h3 class="photo-url">Photo URL</h3>
+                <input type="text" id="edit-photo" name="photo" required>
+              </div>
+            </div>
+          </div>
+          <div class="row">
+            <div class="column-full">
+              <h3>Notes</h3>
+              <textarea id="edit-notes" name="notes" required></textarea>
+              <input class="button edit-button" type="submit" onclick="window.location.href='#entries'" value="SAVE" />
+            </div>
+          </div>
+        </form>
+      </div>
```

Comment:
you want to use the same form for both creating new and editing old entries. This can be removed.